lib/ome/common/module.cpp

```diff
@@ -76,9 +76,9 @@ namespace
     // Global paths (not specific to any component)
     static RegisterModule bin
       ("bin",
-       "BIOFORMATS_BINDIR",
+       "OME_FILES_BINDIR",
```

Comment:
No worries. Rest up, and we can come back to this variable later discussion in follow-up PRs.